From 6b2520cc49edf4d167bec10850b832c308ebf9a3 Mon Sep 17 00:00:00 2001
From: Martin Michlmayr <tbm@cyrius.com>
Date: Sun, 12 Oct 2014 18:20:54 -0400
Subject: [PATCH 3/9] BUILD_WEB_DOCS implies BUILD_DOCS

This fixes the build when -DBUILD_WEB_DOCS=1 is specified wihout
specifying -DBUILD_DOCS.  This fails because man2html/groff are
not found because the find_program command for these programs
is only run if BUILD_DOCS is enabled.
---
 doc/CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/doc/CMakeLists.txt b/doc/CMakeLists.txt
index 96c6d51..46c3f73 100644
--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -27,6 +27,11 @@ endif()
 
 ########################################################################
 
+# BUILD_WEB_DOCS implies BUILD_DOCS
+if (BUILD_WEB_DOCS)
+  set(BUILD_DOCS 1)
+endif()
+
 if (BUILD_DOCS)
   find_program(MAKEINFO makeinfo)
   find_program(TEXI2PDF texi2pdf)
-- 
1.9.3

